external-authority-sources.md

```diff
@@ -0,0 +1,283 @@
+# External sources Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/integration/externalsources**   
```

Comment:
@benbosman : I'm curious your response to this question by @paulo-graca .  I also see this endpoint looks identical to `/api/integration/authorities`.  Is there a reason to separate these external sources into a different endpoint having the same structure? I'm not seeing any difference in the properties returned here, but maybe you had different ideas for this `/externalsources` endpoint that are not yet clear?

Reply:
@paulo-graca @tdonohue 
There were a few reasons for creating a separate endpoint:
* external sources can be imported in authority control or entities (the POST methods). This is not applicable for authority control
* the available features on external sources are not known in advance. For local authority control, it's known the backend supports paging, sorting, total nb hits. For external sources, there's no guarantee for any of these functions to be supported
* The submission UI should only link entities or local authority values to the item (as it happens in DSpace 6), but should never link to an external source without having imported it first. This differentiation may get lost when using a single endpoint

Reply:
I did currently use the name `/externalsources` to avoid confusion with the authorities functionality. This endpoint is also used for entities when no authority control is configured. Naming it `/externalauthorities` may be confusing in a setup where no authority control is used.
I did indeed try to align the contract as much as possible with the `/authorities` endpoint, so from that point of view, it's indeed also possible to use `/externalauthorities`.

So I'd think it depends a bit on the point of view what the preference is, but I can see reasons for either name